app-modules/assist-data-model/src/Filament/Resources/StudentResource/RelationManagers/InteractionsRelationManager.php

```diff
@@ -0,0 +1,49 @@
+<?php
+
+namespace Assist\AssistDataModel\Filament\Resources\StudentResource\RelationManagers;
+
+use Filament\Forms\Form;
+use Filament\Tables\Table;
+use Filament\Infolists\Infolist;
+use Filament\Forms\Components\Hidden;
+use Filament\Forms\Components\Component;
+use Assist\AssistDataModel\Models\Student;
+use Filament\Forms\Components\MorphToSelect;
+use Filament\Resources\RelationManagers\RelationManager;
+use Assist\Interaction\Filament\Resources\InteractionResource\Pages\CreateInteraction;
+use Assist\Interaction\Filament\Resources\InteractionResource\RelationManagers\HasManyMorphedInteractionsRelationManager;
+
+class InteractionsRelationManager extends RelationManager
```

Comment:
We will need to adjust this to extend our `RelationManager` that was recently merged in from Matthews's work.

It ensures that all relation managers have readonly to `false`